PC02_Graffiti_Zeynalova.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Sep  8 20:28:20 2020
+
+@author: fidan
+"""
+
+'''
+Turtle starter code
+ATLS 1300
+Author: Dr. Z
+Author: YOUR NAME
```

Comment:
Forgot to add my name
```suggestion
Author: Fidan
```